Core/Repositories/Device/Device.swift

```diff
@@ -23,7 +23,7 @@ public struct Device: Codable, Equatable {
         type: String = "IOS",
         model: String,
         manufacturer: String = "APPLE",
-        sdkVersion: String,
```

Comment:
Did you specify in the architecture/on the ticket that we should remove the sdkVersion? Just want to make sure Adrian does the same on Android.

Reply:
I updated this in the ticket description and left a comment for Adrain as well, thank you! https://livestyled.atlassian.net/browse/AVS-329